src/components/CryptoCards/types.ts

```diff
@@ -0,0 +1,43 @@
+import { iconTypes } from '../Icon/collection';
+
+export interface CryptoCardProps {
+    /**
+     * A function that will be called if the button is clicked
+     */
+    buttonClickEvent?: () => void;
```

Comment:
let's re-name this to prop to onClick and give it following type
`(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;`

Reply:
done